<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadets</class>
 <widget class="QMainWindow" name="cadets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>790</width>
      <height>540</height>
     </rect>
    </property>
    <widget class="QWidget" name="centralwidget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>770</width>
       <height>500</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #297a00;
color: rgb(41, 122, 0);


</string>
     </property>
     <widget class="QWidget" name="centralwidget_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>671</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 24px; color: white; background-color: #3a9ad9; padding: 10px; border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Автошкола Автолюкс</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scheduleLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 18px; color: white; background-color: #38bd71; padding: 5px; border-radius: 5px;
</string>
         </property>
         <property name="text">
          <string>Список курсантов</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">background-color: white; color: black; border: 1px solid #3a9ad9; border-radius: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="styleSheet">
            <string notr="true">background-color: #3ad97b; color: white; border-radius: 10px; padding: 10px;</string>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="styleSheet">
            <string notr="true">background-color: #d93ad9; color: white; border-radius: 10px; padding: 10px;</string>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="styleSheet">
            <string notr="true">background-color: #d9a63a; color: white; border-radius: 10px; padding: 10px;</string>
           </property>
           <property name="text">
            <string>Выйти</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
